docs/getting_started/trivia_deploying.md

```diff
@@ -65,7 +65,7 @@ Your bot is now live at the URL that Heroku just gave you in the last command ru
 heroku config:set BOTPRESS_PASSWORD=hello123
 ```
 
-### Using Postgres as the database
+### Using Postgres as the database  [[Why make SQL the database by default if it creates and extra step and doesn't work with Heroku which is likely the default host?]]
```

Comment:
Because it doesn't need to be installed (it reduces friction)

Reply:
I'm saying why not make Postgres the default if they are going to host on Heroku?